Plugin/src/main/java/xyz/kyngs/librelogin/common/listener/LoginTryListener.java

```diff
@@ -27,7 +26,7 @@ public LoginTryListener(AuthenticLibreLogin<P, S> libreLogin) {
     }
 
     private void onWrongPassword(WrongPasswordEvent<P, S> wrongPasswordEvent) {
-        if (wrongPasswordEvent.getSource() != AuthenticationSource.LOGIN)
+        if (wrongPasswordEvent.getSource() != AuthenticationSource.LOGIN && wrongPasswordEvent.getSource() != AuthenticationSource.TOTP)
```

Comment:
Here if the authentication source is TOTP, the message that is later sent should be `kick-error-totp-wrong`, not `kick-error-password-wrong`